app/default_overrides.go

```diff
@@ -203,6 +203,9 @@ func (govModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	genState.Params.MaxDepositPeriod = &oneWeek
 	genState.Params.VotingPeriod = &oneWeek
 
+	// the ExpeditedMinDeposit list will always be of size one as set in NewGenesisState in the gov module.
```

Comment:
[not blocking] it would be nice to include in `default_overrides_test.go` a test case for this override so we don't accidentally revert it 😬